rcongui/src/components/collapseCard.js

```diff
@@ -1,40 +1,54 @@
 import React from "react";
-import clsx from 'clsx';
-import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
-import ExpandLessIcon from '@material-ui/icons/ExpandLess'
+import clsx from "clsx";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ExpandLessIcon from "@material-ui/icons/ExpandLess";
 import {
-  IconButton, Card, CardHeader, CardContent,
-  Collapse
+  IconButton,
+  Card,
+  CardHeader,
+  CardContent,
+  Collapse,
 } from "@material-ui/core";
 
+const CollapseCard = ({
+  classes,
+  title,
+  children,
+  onExpand,
+  startOpen = false,
+}) => {
+  const [expanded, setExpanded] = React.useState(false);
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+    onExpand();
+  };
 
-const CollapseCard = ({ classes, title, children, onExpand }) => {
-    const [expanded, setExpanded] = React.useState(false);
-    const handleExpandClick = () => {
-      setExpanded(!expanded);
-      onExpand();
-    };
-  
-    return <Card>
-      <CardHeader title={title} action={
-        <IconButton
-          className={clsx(classes.expand, {
-            [classes.expandOpen]: expanded,
-          })}
-          onClick={handleExpandClick}
-          aria-expanded={expanded}
-          aria-label="show more"
-        >
-          { expanded ? <ExpandLessIcon/> : <ExpandMoreIcon /> }
-        </IconButton>
-      } />
+  React.useEffect(() => {
+    setExpanded(startOpen);
```

Comment:
setting state from a prop could lead to weird bugs I think, i remember that the react docs are talking about that